src/main/com/lunatech/chef/api/persistence/DBEvolution.kt

```diff
@@ -6,5 +6,6 @@ import org.flywaydb.core.Flyway
 object DBEvolution {
     fun runDBMigration(config: FlywayConfig) {
         val flyway = Flyway.configure().dataSource(config.url, config.user, config.password).load()
+        flyway.migrate()
```

Comment:
I see that I removed this line some time ago, but I don't know why. 
Can you please not mix these kind of fixes in non-related PRs? If you detect an issue please create a separate issue and separate PR. It allows to have some traceability in the project which is a good practice :)